app/uk/gov/hmrc/customs/emailfrontend/views/check_your_email.scala.html

```diff
@@ -39,29 +40,34 @@
 
     @helper.form(CheckYourEmailController.submit, Symbol("id") -> "confirmEmailForm", Symbol("novalidate") -> "novalidate") {
         @CSRF.formField
-        @h1_xl(messages("customs.emailfrontend.confirm-your-email.header", email))
 
-        @p(Html(messages("customs.emailfrontend.confirm-your-email.hint")))
+        @fieldset(
```

Comment:
Could you please write a test to check the existence of this component as we are adding a new component in the page? Test (to check this component) in the controller will server the purpose.

Reply:
Can this be added in the next story ?

Reply:
Thats fine Waigwa. Could we please make sure that it has been added in the tech debt?